### PR TITLE
Added Delta456/range

### PR DESCRIPTION
**Pull request by asvvvad**
And added some line breaks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -173,6 +174,8 @@
 
 - [lsp.v](https://github.com/nedpals/lsp.v) - Implementation of the Language Server Protocol spec on V.
 - [setup-v-action](https://github.com/marketplace/actions/setup-v-environment) - GitHub action automation to use V in your workflow.
+- [range](https://github.com/Delta456/range) - Functionality of Python's range() in V.
```
Comment (Delta456) [inline README.md:177]: I think this should be added in a separate section `Utility` .

Comment (alexesprit) [reply README.md:177]: Agree. The `Other` section is for other stuff, not libraries.

Adding the new `Utility` subsection under the `Libraries` section makes more sense.

Comment (alexesprit) [reply README.md:177]: Plus, `range()` could be surrounded by grave accents.

Comment (Delta456) [reply README.md:177]: Yes.

Comment (asvvvad) [reply README.md:177]: I was thinking of the same things but didn't know what to name it
Thanks for the input

Comment (asvvvad) [reply README.md]: Done! Created new category Utility and moved range to it ^^

**Pull request merged**